### PR TITLE
Adds wearable, craftable warning cones

### PR DESCRIPTION
**Pull request by Goldsteel**
A little silly idea I've been throwing around for a while. You can now wear warning cones and make them from 2 plastic. Deconstruct them with wirecutters. They've been made bulky items, since cones are pretty big. The cone sprite has been centred slightly.

image

I am no artist by any stretch and the cone has issues with hairstyles (like a lot of hats do).

image

---------- UPDATE ----------

image

It looks a bit like this now. I'm not sure how I can handle vox without adding another sprite specifically to offset it for dirty birds.

image

You can now wear them as a monkey subtype.

image

Vox specific sprite has been added so you can do beaky cones now.

:cl:
 * rscadd: Adds craftable, wearable warning cones

### CODE REVIEW

**Comment by SonixApache** [comment]
it's too wiiiiiiiiiide

**Comment by Goldsteel** [comment]
If I make it any smaller it just starts overlapping with a bunch of common hairstyles.

**Comment by SonixApache** [comment]
add flags that hide the hair / head, or do some overlay magic maybe?

**Comment by Goldsteel** [comment]
I can try and play around with it but this is the current edition.

**Comment by Intigracy** [comment]
Too wide and off center

**Comment by Goldsteel** [comment]
The only thing left to do now is fix the sprite. I've fixed the off center issue and I'm trying to figure out what width to make it because it doesn't work for other species very well at all.

**Comment by Durgen** [comment]
VLC reference in the item description?

**Comment by ShiftyRail** [comment]
`flags = HIDEHAIRCOMPLETLY` if you want it to completly mask your head.

**Comment by Goldsteel** [comment]
I'm genuinely considering putting a check to see if it's a bird trying to wear it, responding that the cone is the wrong shape for their beaky faces.

**Comment by SonixApache** [comment]
still think it's stupidly wide, but it's cute

**Comment by Goldsteel** [comment]
There isn't much I can do to make it thinner without going outside of the 32x32 sprite box. It overlaps too much with the sprites if I make it thinner. Ideally, I want to make it thinner and raise it up. If anyone has any suggestions about how to make it work, go ahead.

File: code/defines/obj/weapon.dm [hunk 1]
```diff
@@ -587,7 +587,24 @@
...
+	item_state = "cone"
+
+	species_fit = list(VOX_SHAPED)
+
+	flags = HIDEHAIRCOMPLETELY
+	body_parts_covered = FULL_HEAD
+	w_class = W_CLASS_LARGE
+	slot_flags = SLOT_HEAD
+
+obj/item/weapon/caution/attackby(obj/item/I as obj, mob/user as mob)
```
Comment (MadmanMartian) [inline code/defines/obj/weapon.dm]: needs to be /obj/item/weapon, rather than obj/item/weapon

**Pull request merged**